src/aids/admin.py

```diff
@@ -122,11 +131,17 @@ class AidResource(resources.ModelResource):
         attribute="perimeter",
         widget=ForeignKeyWidget('geofr.Perimeter', field="name")
     )
+    programs = fields.Field(
+        column_name="programs",
+        attribute="programs",
+        widget=ManyToManyWidget('programs.Program', field="name")
+    )
 
     class Meta:
         model = Aid
+        exclude = AIDS_EXPORT_EXCLUDE_FIELDS
         # adding custom widgets breaks the usual order
-        export_order = [field.name for field in Aid._meta.fields]
+        export_order = [field.name for field in Aid._meta.fields if field.name not in AIDS_EXPORT_EXCLUDE_FIELDS]  # noqa
 
```

Comment:
Pourrait-t-on écrire `not in exclude` à la place de 'not in AIDS_EXPORT_EXCLUDE_FIELDS' ou la ligne 142 est-elle utilisée ailleurs?

Reply:
j'avais essayé mais ca ne me laisse pas la possibilité :(